{"name": "Логин с неверным логином", "status": "passed", "start": 1719067901513, "stop": 1719067901655, "uuid": "116338ab-0a96-4644-b8cd-54b99f960653", "historyId": "b36fcd0a38afd5ab43d9dc2bb4ff56a5", "testCaseId": "b36fcd0a38afd5ab43d9dc2bb4ff56a5", "fullName": "tests.test_login_user.TestLogin#test_login_with_incorrect_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-JEPQ50G"}, {"name": "thread", "value": "7164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}